Eatery Blue/Eatery Blue/UI/EateryFilter/EateryFilterViewController.swift

```diff
@@ -165,6 +180,13 @@ extension EateryFilterViewController: PaymentMethodsFilterSheetViewControllerDel
         didSelectPaymentMethods paymentMethods: Set<PaymentMethod>
     ) {
         filter.paymentMethods = paymentMethods
+        filter.paymentMethods.forEach { paymentMethod in
+            switch paymentMethod {
+            case .brbs: AppDevAnalytics.shared.logFirebase(BRBFilterPressPayload())
+            case .mealSwipes: AppDevAnalytics.shared.logFirebase(SwipesFilterPressPayload())
+            default: break
```

Comment:
Should it be this?

```suggestion
            default: continue
```

Reply:
Yeah you're right I'll fix that

Reply:
Wait never mind, I think break is right because continue causes a syntax error